package.json

```diff
@@ -1,11 +1,13 @@
 {
     "name": "mathoid-texvcjs",
-    "version": "0.3.2",
+    "version": "1.0.0-alpha1",
```

Comment:
I think we should keep it as `0.x.y` until we are sure/ready to release v1.0, especially because semver is bad with version suffices, so it may interpret `1.0.0-alpha1` as being a higher version than `1.0.0`. Also, the PR's title should be updated accordingly.

Reply:
ok . I was using maven recently. They start counting at 1.0.0-snapshot and I also manged to install the dev version from npm. Do you have any pointes where the -suffix thing fails? Anyway, I'll update the version.